### DISCUSSION TITLE
PD to DC 20V 5A

### DISCUSSION THREAD

**Discussion by Akalikaton** [seerge/g-helper#2292] [General]
Hallo, i am using G14 2020 and i see something just like the Photo. It can tranform PD to DC, when PD100W we can get 20V with 5A. My origin Adapter is 180W, 20V 9A, but i see my Laptop maxmal CPU 40W GPU 60W. Theoretically i can use PD 100W to DC, to get the best Perfermance. 
But i heard about, when battery under 90%, the charging need 150W(20V 7.5A) for a short time. For a normal PD adapter, more than 7A will suddenly shutdown. Is it ture, or how can i solve it with software, lock maxmal 100W.
<img width="365" alt="image" src="https://github.com/seerge/g-helper/assets/127612519/239c3a44-0614-409e-87f9-79e8ce5a7cf3">

**Comment by seerge** [comment]
@Akalikaton hello, I believe this all is handled on hardware level. It's clearly not something an application in windows can control

**Reply by Akalikaton** [reply]
Sad, 90% is 65W PD enough.

**Comment by IceStormNG** [comment]
It depends on the adapter. The barrel plug charger on most laptops have a sense pin which allows the system to check whether it can pull more power or not. If these adapter implement that, the system should realize that it cannot pull more.
If it does not, or the system ignores the sense reading and just expects the factory PSU, the PD Charger, if it is not a cheap chinesium one, will shutdown.
Some of those cheap garbage chargers have no safety and will not shutdown in such cases....

**Reply by Akalikaton** [reply]
Wow, it's not simple just like i thought. Amazing DC has more than 2 Pins.
